src/configure/mod.rs

```diff
@@ -640,6 +673,10 @@ async fn try_smtp_one_param(context: &Context, param: &LoginParam, smtp: &mut Sm
         bail!("could not connect: {}", err);
     }
 
+    if context.shall_stop_ongoing().await {
```

Comment:
this check is not needed anymore with the `async` construction, if this gets triggered we drop the future in the outer `configure` call, which aborts everything.

Reply:
Done